dockerfiles/remote-plugin-go-1.10.7/Dockerfile

```diff
@@ -67,10 +67,11 @@ RUN set -eux; \
     ; \
 	apk del .build-deps \
     ; \
-	mkdir ${HOME}/.cache && chmod -R 777 ${HOME}/.cache \
+	mkdir /.cache && chmod -R 777 /.cache \
```

Comment:
Why does this need to be in root?

Reply:
When launching debugger, it attempted to create .cache directory in the root. Not sure why HOME wasn't resolving properly.

```
/.cache/go-build $ pwd
/.cache/go-build $ ls -la
total 1164
drwxr-xr-x  258 10002400 root          4096 Mar 20 11:28 .
drwxrwxrwx    1 root     root          4096 Mar 20 11:27 ..
drwxr-xr-x    2 10002400 root          4096 Mar 20 11:27 00
drwxr-xr-x    2 10002400 root          4096 Mar 20 11:27 01
drwxr-xr-x    2 10002400 root          4096 Mar 20 11:27 02
```

Reply:
HOME was not set (now it is) so we should put it back to $HOME/.cache

Reply:
@eivantsov ?

Reply:
@benoitf sorry haven't seen your comment. Yes, I'll do that in a new PR then.